src/fides/api/schemas/privacy_experience.py

```diff
@@ -182,71 +183,6 @@ class PrivacyExperienceWithId(PrivacyExperience):
     id: str
 
 
-BinaryChoice = Literal[0, 1]
-
-
-class TCMobileData(FidesSchema):
-    """Pre-parsed TC data and TC string for a CMP SDK:
-
-    https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20CMP%20API%20v2.md#in-app-details
```

Comment:
Just moved to another file, since this is returned now in the both the Privacy Experience and Privacy Preferences responses.